### ISSUE TITLE
text normalizer

### ISSUE THREAD

**Issue by sarpba** [SWivid/F5-TTS#951]
### Checks

- [x] This template is only for feature request.
- [x] I have thoroughly reviewed the project documentation but couldn't find any relevant information that meets my needs.
- [x] I have searched for existing issues, including closed ones, and found not discussion yet.
- [x] I am using English to submit this issue to facilitate community communication.

### 1. Is this request related to a challenge you're experiencing? Tell us your story.

The fine-tuned models have various problems, which could be eliminated by a simple adjustment and normalization of the input text. These problems are the following (in the case of Hungarian, but I think also in the fine-tuned models of other languages).
1, the model does not pronounce numbers
2, the pronunciation of capital letters is problematic due to the lower occurrence in the training dataset
3, the pronunciation of some foreign language words may need adjustment
4, special characters like & # @ % ... all need phonetic transcription
5, time, date issues
...

### 2. What is your suggested solution?

The infer scripts should have a possibility to call the normalizer optimized for their own language into the imput_text channel, before the infer step.

for example there is my hungarian normalizer:
https://github.com/sarpba/F5-TTS_hun/tree/main/src/f5_tts/infer/normalisers/hun

### 4. Can you help us with this feature?

- [ ] I am interested in contributing to this feature.

**Comment by sarpba** [comment]
Here is a wirking gradio with gen_text normalizer
![Image](https://github.com/user-attachments/assets/ae97d571-3e30-4191-b00f-b479b4630e95)
